content/applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst

```diff
@@ -62,59 +55,57 @@ second work center when the first is unavailable for any reason.
-:guilabel:`Resource Time Off` page.
-
-.. image:: work_center_time_off/time-off-button.png
-   :align: center
-   :alt: The Time Off button on the Working Hours pop-up.
+The working hours page displays the standard working hours for the work center. With developer mode
+activated, a :icon:`fa-plane` :guilabel:`Time Off` smart button appears at the top of the page.
+Click it to open the :guilabel:`Resource Time Off` page.
 
-On this page, click :guilabel:`Create` to configure a new time-off entry. On the time-off form, note
+On this page, click :guilabel:`New` to configure a new time-off entry. On the time-off form, note
 the :guilabel:`Reason` for the work center closure (broken, maintenance, etc.), select the affected
 work center as the :guilabel:`Resource`, and choose a :guilabel:`Start Date` and :guilabel:`End
-Date` to specify the period during which the work center will be unavailable. Click :guilabel:`Save`
-and the time off for the work center is logged in Odoo.
+Date` to specify the period during which the work center will be unavailable.
 
 .. image:: work_center_time_off/time-off-form.png
    :align: center
    :alt: The "Resource Time Off" form.
 
-Route orders to an alternative work center
-==========================================
+Alternative work center planning
+================================
 
 Once a work center is within its specified time-off period, work orders sent to it can be
 automatically routed to an alternative work center using the :guilabel:`Plan` button.
 
 Begin by creating a new manufacturing order by selecting :menuselection:`Operations -->
-Manufacturing Orders --> Create`. On the manufacturing order form, specify a :guilabel:`Product`
-that uses the unavailable work center for one of its operations. Click :guilabel:`Confirm` to
-confirm the work order.
+Manufacturing Orders --> New`. On the manufacturing order form, specify a :guilabel:`Product` that
+uses the unavailable work center for one of its operations. Click :guilabel:`Confirm` to confirm the
+work order.
 
 On the confirmed work order, select the :guilabel:`Work Orders` tab. By default, the unavailable
-work center is specified in the :guilabel:`Work Center` column. There is also a green
-:guilabel:`Plan` button on the top left of the page.
+work center is specified in the :guilabel:`Work Center` column. There is also a purple
+:guilabel:`Plan` button at the top left of the page.
 
-.. image:: work_center_time_off/mo-plan-button.png
+.. figure:: work_center_time_off/before-planning.png
    :align: center
-   :alt: The Plan button on a manufacturing order.
+   :alt: Before clicking "Plan", the work order is scheduled at "Main Assembly Line".
 
-Click :guilabel:`Plan` and the work center listed under the :guilabel:`Work Orders` tab is
-automatically changed to the alternative work center.
+   Before clicking "Plan", the work order is scheduled at "Main Assembly Line".
```

Comment:
```suggestion
   In this example, before clicking "Plan", the work order is scheduled at "Main Assembly Line".
```
I just feel like tit should be stated that this is an example being used- I didn't want to add an example block, so this was just an idea.

Reply:
Alternatively, you can make this more vague and less specific, and state that the work center shown is unavailable during this time, and when you click Plan, the work center changes to the alternate one.